pages/newsletter/index.tsx

```diff
@@ -1,60 +1,44 @@
 import styled from "styled-components";
-import { css, StyledSystemProps } from "components/system";
+import { css, all, StyledSystemProps } from "components/system";
 import Head from "components/Head";
 import Link from "components/Link";
 import Box from "components/Box";
 import Flex from "components/Flex";
-import Image from "components/Image";
 import { NewsletterEmailSubscribe } from "components/Newsletter/NewsletterEmailSubscribe";
+import Logo from "components/Logo";
+import { transition } from "components/system";
 
+import engineers from "./assets/engineers.png";
 import webBackground from "./assets/newsletter_web.png";
 import mobBackground from "./assets/newsletter_mob.png";
-import engineers from "./assets/engineers.png";
-import whiteLogo from "./assets/white_logo@1x.png";
 
 const Newsletter = () => {
   return (
     <>
       <Head title="Newsletter" description="Subsribe to our Newsletter!" />
       <StyledBackground
         backgroundImage={[`url(${mobBackground})`, `url(${webBackground})`]}
-        height="100vh"
+        minHeight={["-webkit-fill-available", "100vh"]}
+        backgroundColor="light-purple"
+        backgroundSize="cover"
+        pl={[2, 4]}
+        pr={[2, 4]}
       >
-        <Box mt={[9, 7]} mb={[5, 5]}>
-          <Box as="h1" m={0}>
-            <StyledLink href="/">
-              <Flex
-                justifyContent="center"
-                alignItems="top"
-                height={["24px", "auto"]}
-                width={["120px", "auto"]}
-                css={css({
-                  "&:hover": {
-                    opacity: 0.8,
-                  },
-                })}
-              >
-                <Image
-                  src={whiteLogo}
-                  alt="network infrastructure"
-                  width="100%"
-                />
-              </Flex>
-            </StyledLink>
-          </Box>
+        <Box as="header" mt={[5, 9]} mb={[5, 5]} color="white">
+          <Logo logoWidth={180} logoHeight={38} />
         </Box>
-        <StyledCard>
+
+        <StyledCard as="section">
           <StyledAccessPlane order={[2, 1]}>
-            <StyledBackground
-              backgroundImage={`url(${engineers})`}
-              backgroundSize="100%"
-              width={[282, 534]}
-              height={[158, 300]}
-              backgroundRepeat="no-repeat"
-              ml={[5, -3]}
-              mt={[4, 8]}
-            />
-            <Box>
+            <BoxWrapperStyled>
+              <StyledBackground
+                alignSelf={["center", "initial"]}
+                backgroundImage={`url(${engineers})`}
+                backgroundSize="100%"
```

Comment:
```suggestion
                backgroundSize="cover"
```

Or we don't want to keep proportions?

Reply:
No, in this case, if the background image will not be stretched, then we get a very ugly effect